.github/workflows/publish-google-classroom.yml

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: GC Extractor - Publish
+
+on:
+  workflow_dispatch
+
+jobs:
+  publish-google-classroom-extractor:
+    name: Run unit, style, and type checks
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+
+      - name: Install Python 3.8
+        # very carefully use the tag when testing with act-cli on localhost, commit the hash version
+        # uses: actions/setup-python@v2
+        uses: actions/setup-python@3105fb18c05ddd93efea5f9e0bef7a03a6e9e7df
+        with:
+          python-version: '3.8.x'
+
+      - name: Install Poetry
+        # very carefully use the tag when testing with act-cli on localhost, commit the hash version
+        # uses: Gr1N/setup-poetry@v4
+        uses: Gr1N/setup-poetry@462ac83c852d49e282a1233c4c24c5411696e7c7
+
+      - name: Install Python Dependencies
+        run: python ./eng/build.py install google-classroom-extractor
+
+      - name: Cache Dependencies
+        # very carefully use the tag when testing with act-cli on localhost, commit the hash version
+        # uses: actions/cache@v2
+        uses: actions/cache@0781355a23dac32fd3bac414512f4b903437991a
+        with:
+          path: ~/.cache/pypoetry/virtualenvs
+          key: ${{ runner.os }}-poetry-${{ hashFiles('poetry.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-poetry-
+
+      - name: Run Tests with Coverage
```

Comment:
I agree that we need to re-run some quality checks before publishing, since it is possible to merge something and ignore an error in the trigger on push to `main`. However, perhaps we can simplify a bit by just running `pytest` without capture code coverage? Running type check also helps avoid production bugs, but I wonder if running the linter is really necessary?

Reply:
That makes sense.  I'll update.